test/DebugInfo/TransTypeCompositeCaseClass_.ll

```diff
@@ -13,6 +13,10 @@
 ; RUN: llvm-spirv -r -emit-opaque-pointers %t.spv -o %t.rev.bc
 ; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
 
+; RUN: llvm-spirv --spirv-debug-info-version=nonsemantic-shader-100 %t.bc -o %t.spv
```

Comment:
Should we test something like that in DebugInfo/NonSemantic tests?

Reply:
This test has checks for both OpenCL and NonSemantic versions of the debug info. So IMHO leaving it in the common directory is good enough.